closure/library/BUILD

```diff
@@ -27,6 +27,7 @@ closure_js_library(
         "JSC_INVALID_SUPPRESS",
         "JSC_MISSING_JSDOC",
         "JSC_MISSING_RETURN_JSDOC",
+        "JSC_MISSING_REQUIRE_CALL_WARNING",
```

Comment:
[Optional] Consider filing an issue against Closure Library for this. Suppressions should be for legacy stuff. They shouldn't be introducing new ones.